gamedata/scripts/magazines.script

```diff
@@ -274,12 +274,16 @@ function eject_magazine(weapon)
 	local mag_data = get_mag_data(id)
 	if mag_data then
 		se_mag = alife_create_item(mag_data.section, weapon:parent())
-		local copy = {}
-		copy_table(copy, mag_data)
-		weapon:unload_magazine()
-		set_mag_data(se_mag.id, copy)
-		if #copy.loaded > 0 and weapon_in_slot(weapon) then
-			toggle_carried_mag(se_mag.id)
+		if se_mag then
+			local copy = {}
+			copy_table(copy, mag_data)
```

Comment:
redundant copy? now that get_mag_data copies

Reply:
yeah i can drop this